packages/pluggableWidgets/slider-native/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
     "extends": "@mendix/pluggable-widgets-tools/configs/tsconfig.base",
     "baseUrl": "./",
-    "include": ["./src"]
+    "include": ["node_modules/@mendix/pluggable-widgets-tools/typings", "./src"],
```

Comment:
Shouldn't this be added to the base config? I don't have a clear view on this, but seems out of place here

Reply:
Fixed